### DISCUSSION TITLE
CMI - Add command to remove all homes in one world

### DISCUSSION THREAD

**Discussion by Ergail** [Zrips/CMI#8474] [Enhancement]
Hello, would it be possible to add an admin command to delete all the homes of all the players in a specific world? This could be useful when regenerating resource harvesting worlds.

On Discord it was quickly explained to me that this could be a complex feature to add but I'd rather ask to see if it could be considered one day. I don't know if I should add something on request, if I ever stay available!

**Comment by Momshroom** [comment]
This would be a nice addition

**Comment by CloudeLecaw** [comment]
i agree it would be a nice add.
